### DISCUSSION TITLE
5.2.3 Released

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#4663] [General]
Further production usage of 5.2.2 revealed an issue related to adding new modules to pages - the modules would be added but the permissions would not be set properly. Since this impacted a critical use case, a 5.2.3 release was issued immediately.

# Changes in v5.2.3

#4659 Used RoleName rather than RoleId for consistency in creating permissions @sbwalker
#4658 Added RoleId to Permission Clone method @sbwalker
#4657 Added autocomplete="off" for SMTP Username/Password in Site Setting @thabaum
#4653 Improve scroll position script for non-anchor links @sbwalker

https://github.com/oqtane/oqtane.framework/releases/tag/v5.2.3

**Comment by thabaum** [comment]
Successful upgrade from 5.2.2 to 5.2.3

It was a lot faster, not sure if the payload had anything to do with it but usually it would take a while for example 5.2.1 to 5.2.2 took at least a minute or two before the site refreshed after an upgrade.  This happened time it had happened rather quickly (10 seconds) I am guessing since it was a smaller upgrade.

Thank you @sbwalker for the updates!  Keep 'em rolling out :)

![image](https://github.com/user-attachments/assets/aa35b39b-deb2-466c-ad00-8c57f61f1b04)

**Reply by sbwalker** [reply]
@thabaum the payload is exactly the same as 5.2.2 as the upgrade.zip files are the same size. Either way I am glad the upgrade worked smoothly for you.
